src/uu/cp/src/cp.rs

```diff
@@ -1558,6 +1635,11 @@ fn copy_file(
             println!("{}", context_for(source, dest));
         }
     }
+    let mut copy_debug = CopyDebug {
+        offload: OffloadReflinkDebug::Unknown,
+        reflink: OffloadReflinkDebug::Unsupported,
+        sparse_detection: SparseDebug::No,
+    };
```

Comment:
Is there a relatively easy way where we can do this without giving out a mutable reference? Maybe the `copy_helper` could return a `CopyDebug`?

Reply:
hmm, smart, let me try this :)